University-advisor/ApiCalls.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Net;
+using System.IO;
+using System.Web.Script.Serialization;
+
+namespace University_advisor
+{
+    class ApiCalls {
+
+        public string GetLocationJson(string url) {
+            HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
+            try
+            {
+                WebResponse response = request.GetResponse();
+                using (Stream responseStream = response.GetResponseStream())
```

Comment:
I guess it is better to use implicitly typed variables, so just declare it as var responseSteam.